Dwolla.Client/DwollaClient.cs

```diff
@@ -67,6 +88,14 @@ public class DwollaClient : IDwollaClient
             throw e;
         }
 
+        private static HttpRequestMessage CreateDeleteRequest<TReq>(
+            Uri requestUri, TReq content, Headers headers, string contentType = ContentType)
+        {
+            var r = CreateRequest(HttpMethod.Delete, requestUri, headers);

```

Comment:
Same here. Extract into method that takes the HttpMethod and use it for CreatePostRequest and CreateDeleteRequest